examples/python-stack/README.md

```diff
@@ -1,6 +1,6 @@
 # Datadog CDK Python Example
 
-Use this example Python stack to try out the [datadog-cdk-constructs](https://github.com/DataDog/datadog-cdk-constructs) v2 library. It contains Node, Python and Go Lambda functions.
+Use this example Python stack to try out the [datadog-cdk-constructs](https://github.com/DataDog/datadog-cdk-constructs) v2 library. It contains Node, Python, and Go Lambda functions.
```

Comment:
To address documentation team's comments on https://github.com/DataDog/datadog-cdk-constructs/pull/271